katdal/applycal.py

```diff
@@ -327,7 +327,7 @@ def indirect_cal_product_raw(cache, name, product_type):
             ordered = timestamps.argsort()
             timestamps = timestamps[ordered]
             values = values[ordered]
-            return SimpleSensorData(indirect_cal_product_name(name, product_type),
+            return SimpleSensorGetter(indirect_cal_product_name(name, product_type),
                                     timestamps, values)
```

Comment:
It looks like the indentation is misaligned here.

Reply:
Fixed.